server.js

```diff
@@ -0,0 +1,25 @@
+var dot = require('dotenv').config();
+var server = require('http').createServer(handler);
+var io = require('socket.io')(server);
+var fs = require('fs');
+
+function handler (req, res) {
+  fs.readFile(__dirname + '/index.html', (err, data) => {
+    if (err) {
+      res.writeHead(500);
+      return res.end(`Internal server error ${err}`);
+    }
+    res.writeHead(200);
+    res.end(data);
+  });
+};
+
+server.listen(process.env.SERVER_PORT);
```

Comment:
Add callback function for this method which will ensure that server is connected on specified port and then show the message " Server connected"